ILSpy.AddIn/ILSpy.AddIn.csproj

```diff
@@ -1,10 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
-
-  <PropertyGroup>
-    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
-  </PropertyGroup>
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
```suggestion
<Project>
  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
```

Reply:
📝 The shorthand syntax (where these two lines are combined) implicitly adds two imports:

* It imports **Sdk.props** as the first item in the file
* It imports **Sdk.targets** as the last item in the file

Due to the requirement that **Microsoft.VsSDK.targets** be imported _after_ **Sdk.targets**, we cannot use the shorthand form and must instead explicitly import both **Sdk.props** and **Sdk.targets**.

Reply:
Fixed! OK?

Reply:
Yep!

Reply:
> The shorthand syntax (where these two lines are combined) implicitly adds two imports:
> 
> * It imports **Sdk.props** as the first item in the file
> * It imports **Sdk.targets** as the last item in the file
> 
> Due to the requirement that **Microsoft.VsSDK.targets** be imported _after_ **Sdk.targets**, we cannot use the shorthand form and must instead explicitly import both **Sdk.props** and **Sdk.targets**.

=> This is exactly what the VSIX-SdkProjectAdapter package fixes.